### PR TITLE
[Behat] Move shared storage to behat services

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | yes
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Comment by pamil** [comment]
:+1:, it is not and will not be used anywhere else

File: src/Sylius/Bundle/CoreBundle/Resources/config/test_services.xml [hunk 1]
```diff
@@ -28,8 +27,6 @@
     </parameters>
 
     <services>
-        <service id="sylius.behat.shared_storage" class="%sylius.behat.shared_storage.class%" />
```
Comment (pamil) [inline src/Sylius/Bundle/CoreBundle/Resources/config/test_services.xml]: Seems like it is removed but not defined anywhere else (or I am missing something :)).

Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Resources/config/test_services.xml]: Something went wrong during rebase. Fixed.

**Comment by pjedrzejewski** [comment]
Great @lchrusciel, could you rebase this one? Thanks!

**Comment by lchrusciel** [comment]
@pjedrzejewski done

**Comment by michalmarcinkowski** [comment]
Thanks Łukasz! 👍

**Pull request merged**